apps/console/src/features/users/components/guests/pages/invite-parent-org-user.tsx

```diff
@@ -59,31 +60,43 @@ export const InviteParentOrgUser: FunctionComponent<InviteParentOrgUserPropsInte
         onSubmit,
         [ "data-componentid"]: componentId
     } = props;
+    const dispatch: Dispatch = useDispatch();
+    const userStore: string = PRIMARY_USERSTORE;
 
     const { t } = useTranslation();
-    const { data: allowedRoles } = useRolesList(
-        undefined,
-        undefined,
-        undefined,
-        "users,groups,permissions,associatedApplications"
-    );
 
-    const rolesAutocompleteOptions: RolesAutoCompleteOption[] = useMemo(() => {
+    const {
+        data: groupList,
+        error: groupsError
+    } = useGroupList(userStore, "members", null, true);
+
+    useEffect(() => {
+        if (groupsError) {
+            dispatch(addAlert({
```

Comment:
Are we showing a alert notification in the background while the parent invite modal is open? If yes, it'd be better to show the errors inside the modal window itself IMO.